### PR TITLE
Add ability to filter urls by lastmod

### PR DESCRIPTION
**Pull request by Zak-Bahm**
This commit adds the config option `lastmod` that accepts a timestamp specify the minimum allowable `lastmod` value for urls.
Using this you can filter out older urls that may not be relevant anymore.
This should close this issue: https://github.com/seantomburke/sitemapper/issues/79

### CODE REVIEW

**Comment by nagahshi** [comment]
+1 :100:

**Comment by Zak-Bahm** [comment]
Any updates on this? It should be pretty straight-forward to merge but if there are some issues that need to be fixed I can handle them.

**Comment by seantomburke** [comment]
@Zak-Bahm I think it looks good, just need to resolve the merge conflicts before it can be merged. I believe I tried, but didn't have permission to commit to the branch.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,9 @@ sitemapper.fetch('https://wp.seantburke.com/sitemap.xml')
 
 You can add options on the initial Sitemapper object when instantiating it.
 
++ `requestHeaders`: (Object) - Additional Request Headers
++ `timeout`: (Number) - Maximum timeout for a single URL
```
Comment (seantomburke) [inline README.md:66]: @Zak-Bahm These are duplicated and not needed

**Review by seantomburke — APPROVED** [review]
Looks good, just needs merge conflicts resolved

**Comment by Zak-Bahm** [comment]
I removed the duplicate lines and fixed the conflicts so this should be good to go. I also bumped the version up to 3.2.4 but that can be changed if you think it should be a more major release like 3.3

**Comment by seantomburke** [comment]
@Zak-Bahm Looks like the `package-lock.json` is broken and failing the tests, try removing `package-lock.json` then running `npm install -g npm` and then running `npm install` to fix the `package-lock.json`

**Comment by Zak-Bahm** [comment]
Alright, that should be fixed.

**Comment by Zak-Bahm** [comment]
@seantomburke could you review and approve the changes?

**Comment by seantomburke** [comment]
Looks like it's still failing the tests

**Comment by seantomburke** [comment]
@Zak-Bahm looks like the tests are still failing here, do they pass when you run it locally?

**Comment by Zak-Bahm** [comment]
@seantomburke they were passing when I was running it locally but I wasn't testing older versions of npm like your automated tests do. I've tested back to npm v16.14.15 now so they should all run properly now.

**Comment by Zak-Bahm** [comment]
@seantomburke is there anything on my end preventing this from being merged in?

**Comment by seantomburke** [comment]
@Zak-Bahm There's a merge conflict with the `package-lock.json` file, It can't be merged because of this conflict. I tried fixing it on the `public-square:master` branch, so that would need to be merged into this branch in order to fix it. 
https://github.com/public-square/sitemapper/pull/2

**Comment by seantomburke** [comment]
@Zak-Bahm This is what I see, I'm not able to merge it until the conflicts are resolved.
Screen Shot 2022-07-28 at 11 52 26 PM

**Comment by seantomburke** [comment]
@Zak-Bahm Cloned this PR and fixed the merge conflict. Merged in https://github.com/seantomburke/sitemapper/pull/107, closing this one out!

**Pull request closed without merging**